Package.swift

```diff
@@ -257,5 +257,18 @@ let package = Package(
       ],
       path: "Sources/Examples/RouteGuide/Server"
     ),
+
+    // Client for the PCAP example
+    .target(
+      name: "PCAP",
```

Comment:
Does this target need a clearer name?

Reply:
Sure does. Changed it to 'PacketCapture'